contracts/Colony.sol

```diff
@@ -112,11 +120,52 @@ contract Colony is ColonyStorage {
   function getDomainCount() public view returns (uint256) {
     return domainCount;
   }
-
   function setFunctionReviewers(bytes4 _sig, uint8 _firstReviewer, uint8 _secondReviewer)
   private
   {
     uint8[2] memory _reviewers = [_firstReviewer, _secondReviewer];
     reviewers[_sig] = _reviewers;
   }
+
+  // TODO: Not sure where to put these functions
```

Comment:
I think here is fine, we're at only 170 lines in Colony contract and these feel like Colony core function. Remove todo.